src/hostapi/pulseaudio/pa_linux_pulseaudio_cb.c

```diff
@@ -105,6 +105,34 @@ int PaPulseAudio_updateTimeInfo( pa_stream * s,
     return 0;
 }
 
+/* Release pa_operation always same way */
+int PaPulseAudio_ReleaseOperation(PaPulseAudio_HostApiRepresentation *hostapi,
```

Comment:
I'm not sure whether you should be returning an error if there is a timeout, but right now the code does not return a value so I suggest...

```suggestion
void PaPulseAudio_ReleaseOperation(PaPulseAudio_HostApiRepresentation *hostapi,
```